Cart/Cart/ShoppingCart.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Cart
+{
+    public class ShoppingCart
+    {
+        public readonly List<ICartItem> cartItemList=new List<ICartItem>();
```

Comment:
It should not be public and Cart should own Add and Remove CartItem responsibility